backend/app/api/deps.py

```
@@ -10,21 +10,32 @@
 from app.models.users import Users
 from app.models.shift_assignments import ShiftAssignments
 from app.models.leave_policies import LeavePolicies
+from app.core.security import decode_access_token
 
 oauth2_scheme = OAuth2PasswordBearer(tokenUrl=f"{settings.API_V1_STR}/auth/token")
 
 async def get_current_user(
     token: str = Depends(oauth2_scheme),
     db: AsyncSession = Depends(get_db)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)